integration-tests/scaffolding/helpers.ts

```diff
@@ -1,22 +1,166 @@
+import { ApiRx, Keyring } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { Event } from "@polkadot/types/interfaces";
+import { DispatchError, Event } from "@polkadot/types/interfaces";
 import { Codec, ISubmittableResult } from "@polkadot/types/types";
 import { u8aToHex, u8aWrapBytes } from "@polkadot/util";
-import { pipe, map } from "rxjs";
+import { mnemonicGenerate } from '@polkadot/util-crypto';
+import { pipe, map, tap } from "rxjs";
+import { createKeys } from "./apiConnection";
+import { getAccountInfo, transferFunds } from "./extrinsicHelpers";
 
-export type EventMap = {[key: string]: Event}
-export type Sr25519Signature = {Sr25519: `0x${string}`}
+export const INITIAL_FUNDING = 100000000n;
 
-export function groupEventsByKey() {
+export enum DevAccounts {
+    Alice = "//Alice",
+    Bob = "//Bob",
+    Charlie = "//Charlie",
+    Dave = "//Dave",
+    Eve = "//Eve",
+    Ferdie = "//Ferdie",
+}
+
+export interface AccountFundingInputs {
+    api: ApiRx;
+    amount: bigint;
+    source: string;
+    name?: string;
+    context?: string;
+}
+
+export const generateFundingInputs = (api: ApiRx, context?: string) => ({
+    api,
+    amount: INITIAL_FUNDING,
+    source: DevAccounts.Alice,
+    context
+});
+
+export class EventError extends Error {
+    name: string = '';
+    message: string = '';
+    stack?: string = '';
+    section?: string = '';
+    rawError: DispatchError;
+
+    constructor(source: DispatchError) {
+        super();
+
+        if (source) {
```

Comment:
Can we remove this condition since the source param is not an optional parameter?

Reply:
Sure.